bigbluebutton-html5/imports/api/users-infos/server/handlers/userInformation.js

```diff
@@ -0,0 +1,8 @@
+import { check } from 'meteor/check';
+import addUserInfo from '../modifiers/addUserInfo';
+
+export default function handleUserInformation({ header, body }, meetingId) {
+  check(body, Object);
```

Comment:
Destruct header and body to get the arguments needed. Execute `check` on each, checking for the correct type, then pass to `addUserInfo`